OpenOversight/tests/routes/test_auth.py

```diff
@@ -154,6 +157,10 @@ def test_user_can_register_with_legit_credentials(mockdata, client, session):
         )
 
         assert b"A confirmation email has been sent to you." in rv.data
+        assert (
+            f"{current_app.config['OO_MAIL_SUBJECT_PREFIX']} Confirm Your Account"
+            in log.output.__str__()
```

Comment:
Great additions to these tests! Small nit, the more "pythonic" way of casting to a string would be (same for the rest of the invocations):
```suggestion
            f"{current_app.config['OO_MAIL_SUBJECT_PREFIX']} Confirm Your Account"
            in str(log.output)
```

Reply:
Done!

Reply:
I'm fairly new to 🐍 , so I'm still getting used to the more pythonic way of doing things. Thanks for the note!

Reply:
I have strong opinions about them so I'm happy to provide haha, but also want to respect people's own preferences in that regard too 😛